packages/bp/src/core/messaging/messaging-service.ts

```diff
@@ -144,13 +144,17 @@ export class MessagingService {
     }
 
     const payloadAbsoluteUrl = this.convertToAbsoluteUrls(event.payload)
-    const message = await this.clientsByBotId[event.botId].chat.reply(
+    const message = await this.clientsByBotId[event.botId].messages.create(
       event.threadId!,
-      event.channel,
+      undefined,
       payloadAbsoluteUrl
     )
     event.messageId = message.id
 
+    if (event.payload?.typing) {
+      await new Promise(resolve => setTimeout(resolve, 1000))
```

Comment:
Shouldn't you also check if the event type is typing? And also use the value that might be provided, like we do in channel-web?

Reply:
We don't send typing indicators with builtin payloads anymore, but many clients still send typing indicators manually with custom wait times.